roles/mons/tasks/main.yml

```diff
@@ -31,7 +31,7 @@
 
 
 - name: Start and add that the monitor service to the init sequence
-  service: name=ceph state=started enabled=yes args=mon
```

Comment:
Not sure if this gives some side effects, but using "args=mon" makes systemd trying to start up some "mon.service" instead. Removing it and it works.